### PR TITLE
Add article on PITR process for Gafana (SUP-697)

### PR DESCRIPTION
**Pull request by simonna-k**
Adding help article for PITR process of Grafana for Aiven, so our customer can self service pitr process

### CODE REVIEW

**Comment by harshini-rangaswamy** [comment]
@simonna-k The Vale checks are currently failing because Grafana is being used with a lowercase "g" instead of the expected uppercase "G". Could you please update so that the checks pass successfully?

**Comment by simonna-k** [comment]
> but other already published file uses grafana, not Grafana and correct path is 'grafana', I don' think it will find resource if I spell it as Grafana?

File: docs/products/grafana/howto/pitr-process-for-grafana.rst [hunk 1]
```diff
@@ -0,0 +1,29 @@
+Point-in-time recovery (PITR) process for Aiven for Grafana®
+============================================================
+
+When restore from PITR backup for Grafana, a new service will be created.
```
Comment (harshini-rangaswamy) [inline docs/products/grafana/howto/pitr-process-for-grafana.rst:4]:
```suggestion
The Point-in-Time Recovery (PITR) process allows you to restore your Grafana service using a backup from a specific point in time. When you initiate the restore using the PITR backup for Grafana, a new service will be created to host the restored data. Follow the steps below to perform PITR for Aiven for Grafana:
```

File: docs/products/grafana/howto/pitr-process-for-grafana.rst [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+Steps
+-----
```
Comment (harshini-rangaswamy) [inline docs/products/grafana/howto/pitr-process-for-grafana.rst:7]:
```suggestion

```

Comment (harshini-rangaswamy) [reply docs/products/grafana/howto/pitr-process-for-grafana.rst:7]: I would remove this

File: docs/products/grafana/howto/pitr-process-for-grafana.rst [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+1. Navigate to the backup tab of Grafana service
```
Comment (harshini-rangaswamy) [inline docs/products/grafana/howto/pitr-process-for-grafana.rst:9]: Recommend using the new console navigation while describing steps and for screenshots. The old screen (with top nav tabs) will be phased out soon. 

I would rewrite the above line as follows: In the Aiven for Granfana Service, select Backups from the left sidebar.

File: docs/products/grafana/howto/pitr-process-for-grafana.rst [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+2. Click 'Fork & restore'
```
Comment (harshini-rangaswamy) [inline docs/products/grafana/howto/pitr-process-for-grafana.rst:11]:
```suggestion
2. Click 'Fork & restore'.
```

File: docs/products/grafana/howto/pitr-process-for-grafana.rst [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+4. Once you have confirmed all the relevant selections, scroll down to the bottom of the popup and click 'Create fork' 
+
+5. 'Overview' tab will be loaded for the new forked service, the service status will be seen as 'Rebuilding' until the node comes up 
+
+.. image:: /images/products/grafana/grafana-pitr-after-fork.png
+    :alt: restore is rebuilding after clicking 'create fork'
+
+.. Tip::
+    Service may have more than 1 backup file. Make sure to select the correct PITR backup from the dropdown list 'Select backup to fork from'.
```
Comment (harshini-rangaswamy) [inline docs/products/grafana/howto/pitr-process-for-grafana.rst:29]: I think this tip is not required here.

File: docs/products/grafana/howto/pitr-process-for-grafana.rst [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+    :alt: click 'Fork & restore' from backup tab of Grafana service from Aiven console
+
+3. After you have clicked the button, a new popup will appear to change specifics for the service restore
+
+.. image:: /images/products/grafana/grafana-pitr-new-db-fork-popup.png
+    :alt: popup for setting specifics of the service restore.
+
+4. Once you have confirmed all the relevant selections, scroll down to the bottom of the popup and click 'Create fork' 
+
+5. 'Overview' tab will be loaded for the new forked service, the service status will be seen as 'Rebuilding' until the node comes up
```
Comment (harshini-rangaswamy) [inline docs/products/grafana/howto/pitr-process-for-grafana.rst:23]:
```suggestion
5. You will be redirected to the **Overview** page of the newly forked service. The service is in the **Rebuilding** status while it is being created. Once the service is ready, the status changes to **Running**. 
```

File: docs/products/grafana/howto/pitr-process-for-grafana.rst [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+-----
+
+1. Navigate to the backup tab of Grafana service
+
+2. Click 'Fork & restore'
+
+.. image:: /images/products/grafana/grafana-pitr-fork-restore.png
+    :alt: click 'Fork & restore' from backup tab of Grafana service from Aiven console
+
+3. After you have clicked the button, a new popup will appear to change specifics for the service restore
```
Comment (harshini-rangaswamy) [inline docs/products/grafana/howto/pitr-process-for-grafana.rst:16]:
```suggestion
3. In the **New Database Fork** screen, 

     - Provide a name for the new service.
     - Verify that the appropriate Project name is chosen.
     - Select the desired backup from the available options using the drop-down list.
      
       .. Tip::
            If your service has multiple backup files, ensure that you select the correct PITR backup from the dropdown list. 
     - Choose your cloud provider, preferred cloud region, and the service plan accordingly.
    
```

File: docs/products/grafana/howto/pitr-process-for-grafana.rst [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+
+.. image:: /images/products/grafana/grafana-pitr-new-db-fork-popup.png
+    :alt: popup for setting specifics of the service restore.
+
+4. Once you have confirmed all the relevant selections, scroll down to the bottom of the popup and click 'Create fork'
```
Comment (harshini-rangaswamy) [inline docs/products/grafana/howto/pitr-process-for-grafana.rst:21]:
```suggestion
4. Click **Create fork** to create the new forked service.
```

**Review by harshini-rangaswamy — APPROVED** [review]
Thanks for adding this content! LGTM.

**Pull request merged**